src/chapter_3/PDAs.md

```diff
@@ -0,0 +1,315 @@
+# PDAs
+Program derived addresses are addresses with special properties.
+
+Unlike normal addresses, PDAs are not public keys and therefore do not have an associated private key. There are two use cases for PDAs. They provide a mechanism to build hashmap-like structures on-chain and they allow programs to sign instructions.
+
+Working with PDAs is one of the most challenging parts of working with Solana.
+This is why in addition to our explanations here, we want to provide you with some further resources.
+We cover everything there is to know about PDAs here but it may be useful to learn about PDAs from different perspectives.
```

Comment:
Should probably move these links to the end of the chapter.

Reply:
I put them at the front because if a reader becomes frustrated with my writing they might not read to the end.

Reply:
counterargument is that adding the links here as unnecessary information adds to the frustration. better to get straight to the point imo